### PR TITLE
Creators improve descriptions

### PR DESCRIPTION
**Pull request by Sponge96**
## Changelog Description
Slight tweaks to create saver descriptions.

## Additional info
We had some user confusion between image and render creators so hope to address that by improving the descriptions slightly.

## Testing notes:
1. open AYON Publisher Create tab
2. select Image (saver) and view the new description
3. select Render (saver) and view the new description

### CODE REVIEW

File: client/ayon_fusion/plugins/create/create_saver.py [hunk 1]
```diff
@@ -18,12 +18,12 @@ class CreateSaver(GenericCreateSaver):
...
     name = "render"
     product_type = "render"
-    description = "Fusion Saver to generate image sequence"
+    description = "Fusion Saver to generate a multi-file image sequence"
 
     default_frame_range_option = "current_folder"
 
     def get_detail_description(self):
-        return """Fusion Saver to generate image sequence.
+        return """Fusion Saver to generate a multi-file image sequence.
```
Comment (BigRoy) [inline client/ayon_fusion/plugins/create/create_saver.py:26]: I like this less personally - it's overly specific but also untrue. I'm pretty sure the `render` product type is completely fine to be a single frame. I do like the adjustment on the description of the image saver though.

@Sponge96 can you revert this one?

Comment (BigRoy) [reply client/ayon_fusion/plugins/create/create_saver.py:26]: Or actually - are you ok with reverting it? if so, let me push something to this branch too.

**Comment by BigRoy** [comment]
Lovely - I'll just push on other fix.

File: client/ayon_fusion/plugins/create/create_saver.py (hunk 1 above)
Comment (Sponge96) [reply client/ayon_fusion/plugins/create/create_saver.py]: I'll see how strong people feel on my end, I've reverted for now

**Comment by BigRoy** [comment]
Pushed https://github.com/ynput/ayon-fusion/pull/19/commits/bde3904baadde203026a00381038f466504bd676

Before fix:

image
image


After fix:

image
image

**Comment by BigRoy** [comment]
Ok like this @Sponge96 ?

**Comment by Sponge96** [comment]
> Ok like this @Sponge96 ?

@BigRoy Yeah lets go with just this for now, can always make another PR if they feel strongly about changing the other

**Pull request merged**